.circleci/config.yml

```diff
@@ -251,9 +251,4 @@ workflows:
             - ubuntu_22_04   # LTS
             - vscode_stable
             - wasm
-      - docs_deploy:
-          requires:
-            - release
-          filters:
-            branches:
-              only: master
+
```

Comment:
If we weren't about to delete this file, I'd be more concerned about the blank line at the end. 😉